### PR TITLE
Introduction of `val` and added some docstrings for pos, ang, val

### PR DESCRIPTION
**Pull request by Wikunia**
Ability to use arbitrary returns as explained in #62 

Example:
```
javis(video, [
      BackgroundAction(1:50, :framenumber, ground),
      Action(:same, (args...)->text(val(:framenumber), Point(0, -200))),
], pathname="test.gif")


function ground(video, action, framenumber) 
    background("white")
    sethue("blue")
    return "Frame: $framenumber"
end
```

### CODE REVIEW

**Comment by Wikunia** [comment]
I assume the current problem is having a different font for different systems. 
Using LaTeX will probably fix this

File: src/Javis.jl [hunk 1]
```diff
@@ -491,28 +500,49 @@ function perform_transformation(trans::InternalRotation)
     rotate(trans.angle)
 end
 
-pos(p::Point) = p
-pos(t::Transformation) = t.p
+"""
+    val(s::Symbol)
+
+Get access to the value that got saved in `s` by a previous action.
+If you want to access a position or angle check out [`pos`](@ref) and [`angle`](@ref).
```
Comment (TheCedarPrince) [inline src/Javis.jl:507]: Similarly to what we discussed with `ang` and `pos` in #66, can you also give an alternative name like `get_value`? Thanks!

File: src/Javis.jl [hunk 2]
```diff
@@ -491,28 +500,49 @@ function perform_transformation(trans::InternalRotation)
...
-end
+"""
+    angle(s::Symbol)
+
+Get access to the angle that got saved in `s` by a previous action.
+
+# Returns
+- `Float64`: the angle stored by a previous action i.e via `return Transformation(p, angle)`
+"""
+angle(s::Symbol) = angle(val(s))
```
Comment (TheCedarPrince) [inline src/Javis.jl:545]: Merge conflict with #66 based on naming -- just a heads-up for you to be aware of.

File: src/Javis.jl [hunk 3]
```diff
@@ -695,7 +725,7 @@ end
 export javis, latex
 export Video, Action, BackgroundAction, Rel
 export Line, Translation, Rotation, Transformation
-export pos, angle
+export val, pos, angle
```
Comment (TheCedarPrince) [inline src/Javis.jl:728]: If you add the alternative name for `val`, don't forget to add it here.

File: test/animations.jl [hunk 4]
```diff
@@ -110,13 +110,19 @@ end
     end
 end
 
+function disp_text(t, p)
+    setfont("Helvetica", 24)
+    settext(t, p)
+end
+
```
Comment (TheCedarPrince) [inline test/animations.jl:117]: Error here due to not all systems possibly possessing this font. Not sure how to get around this.

File: src/Javis.jl (hunk 1 above)
Comment (Wikunia) [reply src/Javis.jl:507]: I'll do this in #66 and merge that one first then.

File: src/Javis.jl [hunk 5]
```diff
@@ -491,28 +500,49 @@ function perform_transformation(trans::InternalRotation)
...
+
+"""
+    pos(s::Symbol)
+
+Get access to the value that got saved in `s` by a previous action.
+If you want to access a position or angle check out [`pos`](@ref) and [`angle`](@ref).
+
+# Returns
+- `Point`: the point stored by a previous action.
+"""
```
Comment (Wikunia) [inline src/Javis.jl:532]: stupid wrong copy paste mistake

File: test/animations.jl (hunk 4 above)
Comment (Wikunia) [reply test/animations.jl]: I just changed it to setting the radius of a circle instead.

File: src/Javis.jl (hunk 2 above)
Comment (Wikunia) [reply src/Javis.jl]: Yeah it was a hard one but I think I managed it.

**Pull request merged**